### PR TITLE
Update slivar recipe to include pslivar

### PR DESCRIPTION
**Pull request by williamrowell**
Adds `pslivar` binary to recipe for `slivar` package.

As requested in https://github.com/brentp/slivar/issues/73, this update will add the v0.1.12 `pslivar` binary.
I followed the format recommended in [this thread](https://github.com/bioconda/bioconda-recipes/issues/4772) for adding a second download to a recipe.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by williamrowell** [comment]
@BiocondaBot please add label

File: recipes/slivar/meta.yaml [hunk 1]
```diff
@@ -11,13 +11,15 @@ source:
...
-  number: 0
+  number: 1
   skip: True  # [osx]
 
 requirements:
   run:
     - htslib
     - zlib
+    - coreutils
+    - curl
```
Comment (dpryan79) [inline recipes/slivar/meta.yaml:22]: Does it need curl during runtime? It looks like this could just be in the `build` section.

**Review by dpryan79 — CHANGES_REQUESTED** [review]

File: recipes/slivar/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/slivar/meta.yaml:22]: Actually, the binaries are statically linked, so I expect there doesn't need to be anything in the `run` section.

Comment (williamrowell) [reply recipes/slivar/meta.yaml:20]: Makes sense.  I was making minimal changes to existing recipes.  I removed the run requirements and shifted `coreutils` and `curl` to build requirements.

**Review by brentp — APPROVED** [review]

**Comment by brentp** [comment]
ping @dpryan79

**Comment by williamrowell** [comment]
@brentp updated slivar to v0.1.13, so I bumped the versions and updated the sha256sums so I won't need to submit a new PR for review again immediately.

**Review by brentp — APPROVED** [review]

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

Thanks everyone!

**Pull request merged**